modules/process_incoming_text.py

```diff
@@ -0,0 +1,86 @@
+import logging
+import string
+
+from texter import send_text
+from utils import quote, date_is_valid, date_string_to_date
+from i18n import msg_subscribe, msg_unsubscribe, msg_placeholder_child, msg_failure, \
+                    msg_failed_date, subscribe_keywords
+
+class TextProcessor(object):
+    def process_subscribe(self, keyword, child_name, date, language):
+        # TODO: Store data in the system
+        return child_name + msg_subscribe(language)
```

Comment:
There should be a space between the child's name and the message subscription since there isn't one at the beginning of what msg_subscribe() returns.